src/site/xdoc/index.xml

```diff
@@ -15,346 +15,122 @@
-              </p>
-            </div>
-          </subsection>
-
-          <subsection name="Proposal 10">
-            <div id="james-logo-10-preview">
-              <p><img src="logo-call/james-logo-10.png"/></p>
-              <p>
-                <a href="#">Click to change the upper left logo on this page and preview this proposal.</a>
-              </p>
-            </div>
-          </subsection>
-
-        </div>
-
-<!-- 
-        <div id="tabs-5">
-  
-              <object width="480" height="385">
-                <param name="movie" value="http://www.youtube.com/v/0Mx7qy7i3Hs?fs=1&amp;hl=en_US"></param>
-                <param name="allowFullScreen" value="true"></param><param name="allowscriptaccess" value="always"></param>
-                <embed src="http://www.youtube.com/v/0Mx7qy7i3Hs?fs=1&amp;hl=en_US" type="application/x-shockwave-flash" allowscriptaccess="always" allowfullscreen="true" width="480" height="385">
-                </embed>
-              </object>
-  
-        </div>
--->
+
+          <subsection name="Software Components">
+
+              <p>Developers looking for a modular mail platform on which to build can look
+                 at the modules and libraries used to compose James Server 3.0.</p>
+
+              <p><a href='http://james.apache.org/imap/'>Apache James IMAP</a> provides a flexible codec for IMAP,
+                 command processors and a sample data access layer. In combination with a socket layer,
+                 and a mailbox persistence, this library can be used to create an IMAP server.</p>
+
+              <p>The <a href='http://james.apache.org/mailets/'>Apache James Mailet project</a> collects products
+                 related to mailets (mail processing components analogous to servlets). These are independent of the
+                  <a href='http://james.apache.org/server/'>James server</a> and can be reused in any mailet container.</p>
+
+              <ul>
+                <li>The <a href='http://james.apache.org/mailet/api'>Apache James Mailet API</a> specifies mailets.</li>
+                <li>The <a href='http://james.apache.org/mailet/base'>Apache James Mailet Basic Toolkit</a>
+                    collects utilities and lightweight frameworks useful to develop and test mailets.</li>
+                <li><a href='http://james.apache.org/mailet/standard'>Apache James Standard Mailets</a> collects
+                    general processing mailets with limited dependencies.</li>
+                <li><a href='http://james.apache.org/mailet/crypto'>Apache James Crypto Mailets</a> collects
+                    mailets which perform cryptographic processing such as signing, encrypting,
+                    decrypting and signature verification.</li>
+              </ul>
+
+              <p><a href='http://james.apache.org/protocols/'>Apache James Protocols</a> project delivers a lightweight,
```

Comment:
Protocols is comming with several implementations of popular mail protocols like : 
 - IMAP
 - SMTP
 - POP3
 - ManageSieve

I would find it helpfull to have a list of implemented protocols on the homepage....